tycho-plugins/repository-utils/src/main/java/org/jboss/tools/tycho/sitegenerator/GenerateRepositoryFacadeMojo.java

```diff
@@ -91,15 +91,18 @@
 	}
 
 	public static final Set<String> defaultSystemProperties = new HashSet<String>(Arrays.asList(new String[] {
+		// these are all parameters of the Jenkins job; if not set they'll be null
 		"BUILD_ALIAS",
 		"JOB_NAME",
 		"BUILD_NUMBER",
 		"BUILD_ID",
-		"HUDSON_SLAVE",
 		"RELEASE",
 		"ZIPSUFFIX",
 		"TARGET_PLATFORM_VERSION",
 		"TARGET_PLATFORM_VERSION_MAXIMUM",
+		
+		// these are environment variables so should be valid when run in Jenkins or for local builds
+		"HOSTNAME", // replaces HUDSON_SLAVE
```

Comment:
Jenkins also set the NODE_NAME ( https://wiki.jenkins-ci.org/display/JENKINS/Building+a+software+project#Buildingasoftwareproject-JenkinsSetEnvironmentVariables ). Which one do you prefer?